Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationService.cs

```diff
@@ -211,7 +212,7 @@ public override void Update()
                     break;
             }
 
-            if (profile.IsCameraControlEnabled)
+            if (profile.IsCameraControlEnabled && !XRDevice.isPresent)
```

Comment:
Please add comment explaining why we do this. Note that I believe real reason identified by @keveleigh is that the camera movement is mapped to AXIS_3, and AXIS_3 happens to be the select trigger on WMR controllers